dashboard/app/controllers/scripts_controller.rb

```diff
@@ -45,6 +45,16 @@ def show
     @section = current_user&.sections&.find_by(id: params[:section_id])&.summarize
     sections = current_user.try {|u| u.sections.where(hidden: false).select(:id, :name, :script_id, :course_id)}
     @sections_with_assigned_info = sections&.map {|section| section.attributes.merge!({"isAssigned" => section[:script_id] == @script.id})}
+
+    if current_user.levelbuilder?
+      @script.stages.each do |stage|
+        next unless stage.visible_after
+
+        formatted_time = Time.parse(stage.visible_after).strftime("%I:%M %p %A %B %d %Y %Z")
+        num_days_away = ((Time.parse(stage.visible_after) - Time.now) / 1.day).ceil.to_s
+        flash[:notice] = "The lesson #{stage.name} will be visible after #{formatted_time} (#{num_days_away} Days)"
```

Comment:
please also test the following scenarios:
1. the banner is not visible when `visible_after` is absent
2. the banner is not visible if the time indicated by `visible_after` has passed (this may be easier to implement after you've added the code to actually hide the lesson, so ok to do in a later PR)
3. we should do something reasonable if 2 lessons are hidden. if the current code means that both banners appear, that's great. but if one replaces the other, then we might need to do something different -- anything that will prevent a nasty surprise where a lesson remains hidden when we need it to be visible.
4. no banner is visible for students

Reply:
also, what happens if two lessons are hidden -- do we show two banners?